daemon/openrazer_daemon/misc/ripple_effect.py

```diff
@@ -30,7 +30,9 @@ def __init__(self, parent, device_number):
         self._shutdown = False
         self._active = False
 
-        self._kerboard_grid = KeyboardColour()
+        self._matrixdims = self._parent._parent.MATRIX_DIMS
```

Comment:
I would of done something like `self._rows, self._cols = self._parent._parent.MATRIX_DIMS` but doesnt make much difference.